### DISCUSSION TITLE
Need example PGP/MIME signed mail that may be published

### DISCUSSION THREAD

**Discussion by silbe** [RotherOSS/otobo#2400] [Q&A]
Would it be possible for someone to create an example PGP/MIME signed email generated by OTOBO that can be shared in public (public bug trackers, public git repos, etc.)?

An organisation I'm communicating with is using OTOBO and signing the emails using the PGP/MIME format. The mails verify fine in one MUA but not in another. I need an example mail that triggers the issue and can be shared publicly so the developers of the MUA and of the libraries it uses can use it for debug and test purposes. The OTOBO instance used by the organisation (and probably hosted by someone else) includes quite a bit of personal data so the original mails cannot be shared publicly.

**Comment by svenoe** [comment]
Hi Sascha,

just a short note, that our [forums](https://otobo.de/de/forums/) are probably a better location for questions like this.

Best regards, Sven
